src/codegen/index.ts

```diff
@@ -206,9 +219,21 @@ export default class LLVMCodeGen {
   public genCallExpression(node: ts.CallExpression): llvm.Value {
     const name = node.expression.getText();
     const args = node.arguments.map(item => {
-      return this.genExpression(item);
+      if (item.kind === ts.SyntaxKind.Identifier) {
+        return this.genAutoDereference(this.genExpression(item));
+      } else {
+        return this.genExpression(item);
+      }
     });
-    const func = this.module.getFunction(name)!;
+    let func: llvm.Function;
+    switch (name) {
+      case 'console.log':
```

Comment:
Should `console` be considered a module?

Reply:
When we go into the stage for implementing the build in functions, consider this question again. Now this is enough.